### PR TITLE
[Draft] Update EdgeDB Docs

### PR DESCRIPTION
**Pull request by ajwootto**

### CODE REVIEW

File: docs/extras/edgedb.md [hunk 1]
```diff
@@ -132,23 +131,6 @@ In this guide we explored:
...
-
-This data can be imported to DevCycle and EdgeDB in a number of different methods:
-- Via the [EdgeDB API](/best-practices/edgedb-and-edge-flags/edge-db-via-api-simple)
-- Through the [Zapier Integration](/best-practices/edgedb-and-edge-flags/import-from-anywhere)
-
-Once a Super Segment has been created in EdgeDB it can then be accessed via Feature Flags using [Custom Properties](/extras/advanced-targeting/custom-properties) within a targeting rule.
-
-Make sure to add the Custom Property to any feature, using the same key used when creating the Super Segment via the API or Zapier Integration.
-
 # Data Residency
```
Comment (cdruxerman) [inline docs/extras/edgedb.md:134]: Should we take this out since we never really sorted how we would do it?

**Review by cdruxerman — COMMENTED** [review]
check out inline comments

File: docs/extras/edgedb.md [hunk 2]
```diff
@@ -1,22 +1,17 @@
...
-
-# Edge Flags
-EdgeDB is a low-latency, centralized user data storage system that is accessible across all of our SDKs. With EdgeDB, you can accurately use targeting rules without having to pass all of a user’s data into the SDKs each time. Once targeting data is stored in EdgeDB, it can be easily accessed on any platform without the need for data pipelines.
+To learn more about EdgeDB check out the documentation for the features powered EdgeDB below.
 
-This guide will explain how to set up and use EdgeDB. It will also discuss some use cases to help you determine where to implement EdgeDB in your project.
+# Stored Custom Properties
+This guide will explain how to set up and use EdgeDB to target users using Stored Custom Properties. It will also discuss some use cases to help you determine where to implement EdgeDB in your project.
 
 ### Prerequisites
```
[the remark this answers is not in the document]
Comment (ajwootto) [reply docs/extras/edgedb.md:17]: which respective areas do you mean?

[the remark this answers is not in the document]
Comment (cdruxerman) [reply docs/extras/edgedb.md:19]: add links to the bullet points below. 

- identifying users (https://docs.devcycle.com/sdk/features#identifying-a-user-or-setting-properties)
- targeting rules (https://docs.devcycle.com/essentials/targeting)
- custom properties (https://docs.devcycle.com/extras/advanced-targeting/custom-properties)

**Review by cdruxerman — APPROVED** [review]

**Pull request merged**